charts/argo-cd/templates/redis/_configs.tpl

```diff
@@ -0,0 +1,29 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Why do we need extra tpl file when templates are used used just once inside the ConfigMap and templating contains only redis container port? This can be placed inline with:

```yaml
data:
  redis_liveness.sh: |
    response=$(
      redis-cli \
        -h localhost \
        -p {{ .Values.redis.containerPorts.redis }} \
        ping
    )
    if [ "$response" != "PONG" ] && [ "${response:0:7}" != "LOADING" ] ; then
      echo "$response"
      exit 1
    fi
    echo "response=$response"
```

Reply:
Hi @pdrastil 
I simply tried to stick to the same logic as in the redis-ha helm chart from DandyDeveloper.
See here: https://github.com/DandyDeveloper/charts/blob/master/charts/redis-ha/templates/_configs.tpl
If you prefer, I can move that to the ConfigMap as you proposed.

Reply:
I'd also prefer to have it directly inside the ConfigMap (for simplicity). I think that the HA helm chart reuses the "script" depending on the mode you install it.

Reply:
Hi,
I rebased my branch and moved the probes scripts directly into the ConfigMap.

Reply:
Sorry, I had forgotten to bump the version.
Looks like someone needs to re-trigger the CI jobs :)
Thanks!